react/features/notifications/components/native/Notification.tsx

```diff
@@ -0,0 +1,302 @@
+                    <View pointerEvents = 'box-none'>
+                        { this._renderContent() }
+                    </View>
+                    <View style = { styles.btnContainer }>
+                        { this._mapAppearanceToButtons() }
+                    </View>
+                </View>
+                <IconButton
+                    color = { BaseTheme.palette.icon04 }
+                    onPress = { this._onDismissed }
+                    src = { IconCloseLarge }
+                    type = { BUTTON_TYPES.TERTIARY } />
+            </View>
+        );
+    }
+
+    /**
+     * Renders the notification's content. If the title or title key is present
+     * it will be just the title. Otherwise it will fallback to description.
+     *
+     * @returns {Array<ReactElement>}
+     * @private
+     */
+    _renderContent() {
+        // @ts-ignore
+        const { icon, maxLines = DEFAULT_MAX_LINES, t, title, titleArguments, titleKey, concatText } = this.props;
+        const titleText = title || (titleKey && t(titleKey, titleArguments));
+        const description = this._getDescription();
+        const titleConcat = [];
+        const descriptionStyles = icon === NOTIFICATION_ICON.PARTICIPANTS
+            ? styles.contentTextInteractive : styles.contentText;
+
+        if (concatText) {
+            titleConcat.push(titleText);
+        }
+
+        if (description?.length) {
+            return [ ...titleConcat, ...description ].map((line, index) => (
+                <Text
+                    key = { index }
+                    numberOfLines = { maxLines }
+                    style = { descriptionStyles }>
+                    { replaceNonUnicodeEmojis(line) }
+                </Text>
+            ));
+        }
+
+        return (
+            <Text
+                numberOfLines = { maxLines }>
+                { titleText }
+            </Text>
+        );
+    }
+
+    /**
+     * Renders the avatar.
+     *
+     * @returns {Array<ReactElement>}
+     * @private
+     */
+    _renderAvatar() {
+        // @ts-ignore
+        const { _participants } = this.props;
+
+        return (
+            <>
+                {
+                    _participants?.map((p: any, index: number) => (
+                        <View
+                            key = { index }
+                            style = { styles.avatarContainer }>
+                            <Avatar
+                                displayName = { p.name }
+                                participantId = { p.id }
+                                size = { 32 } />
+                            <Text
+                                numberOfLines = { 1 }
+                                style = { styles.avatarText }>
+                                { p.name }
+                            </Text>
+                        </View>
+                    ))
+                }
+            </>
+        );
+    }
+
+    _getDescription: () => Array<string>;
```

Comment:
I don't think this is necessary now that this is TS.

Reply:
Will check if it is so.